### ISSUE TITLE
Librerías Prohibidas 

### ISSUE THREAD

**Issue by dgonzalezg** [IIC2513-1/Syllabus#1]
## Las siguientes librerías quedan prohibidas para el proyecto semestral.
- [MUI](https://mui.com/)
- [Bulma](https://bulma.io/)
- [React Session](https://github.com/grizzthedj/react-session)
- [Auth0](https://auth0.com/es)
- [Headless UI](https://headlessui.dev/)
- [Bootstrap](https://getbootstrap.com/)

Pueden responder a esta Issue preguntando si pueden usar alguna librería.

**Comment by BenjaRubio** [comment]
Se puede usar redux?

**Comment by IsaacConDosA** [comment]
> Se puede usar redux?

Sí. Aunque quizás es mucho para el alcance del proyecto. Si quieres tener estados globales dentro de tu app, te recomiendo el nativo `useContext`

**Comment by Kevinnevik** [comment]
Hola, se puede usar el siguiente link para el proyecto https://tailwindui.com/?

**Comment by IsaacConDosA** [comment]
> Hola, se puede usar el siguiente link para el proyecto https://tailwindui.com/?

No pueden ocupar ninguna librería que provea componentes listos para usar. Todos los componentes y su estilo lo deben hacer ustedes.

**Comment by nicolascruzi** [comment]
Se puede usar socket.io? para el chat

**Comment by IsaacConDosA** [comment]
> Se puede usar socket.io? para el chat

Sí, puedes usarla. Si no logras implementar lo del chat en tiempo real, háblalo con tu ayudante y se puede flexibilizar.

**Comment by Kevinnevik** [comment]
Podemos usar cloudinary. para guardar informacion que sube el cliente?
